README.md

```diff
@@ -1,2 +1,7 @@
 # ci-oracle-client-runner
-A container that contains an oracle client for running against Concourse 
+A container that contains an oracle client for running against Concourse.\
+The files here are based on the OracleInstantClient docker images provided by Oracle, that can be found here: https://github.com/oracle/docker-images/tree/master/OracleInstantClient \
+This is based on the Oracle Linux 7-slim (dist-amd64) container image: https://github.com/oracle/container-images/tree/573754fe56e7e8d570c554e8152fc9606c89561d/7-slim
+
+The Dockerfile is based on Oracle template: https://github.com/oracle/docker-images/blob/master/OracleInstantClient/dockerfiles/19/Dockerfile
```

Comment:
As above, there's a mutable reference in the URL here which means the page contents will change over time.

Reply:
Also updated.